packages/app-bridge/src/repositories/HubRepository.ts

```diff
@@ -0,0 +1,77 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { HttpClient, rgbObjectToRgbString, rgbStringToRgbObject } from '../utilities';
+
+/**
+ * @deprecated
+ */
+export const getHub = async (portalId: number): Promise<Record<string, unknown>> => {
+    const { result } = await HttpClient.get(`/api/hub/settings/${portalId}`);
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    //@ts-ignore
+    return result.hub;
+};
+
+/**
+ * @deprecated
+ */
+export const updateHub = async (portalId: number, data: Record<string, unknown>): Promise<void> => {
+    await HttpClient.post(`/api/hub/settings/${portalId}`, data);
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const mapHubApiToFlatSettingsDottedNotation = (hubSettings: any) => ({
+    'brandhome_appearance.grid.width': hubSettings?.brandhome_settings?.document_appearance?.grid?.width,
+    'brandhome_appearance.grid.margin': hubSettings?.brandhome_settings?.document_appearance?.grid?.margin,
```

Comment:
Using `Record<string, unknown>` would require to do `'document_appearance' in hubSettings?.brandhome_settings` on every single fields and recursively because `document_appearance` children are also `unknown`.

![Screenshot 2023-03-09 at 16 10 11](https://user-images.githubusercontent.com/6846950/224066860-38be4957-625a-4b90-932e-bf80f715af89.png)